docs/game/cheat_sheet.md

```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
 Manners
 -------
 
-Some general manners guidelines for panel games, and specifically Just A Minute
+Panel games, much like the greatness of Great Britain, are founded on
+good manners. Here are some guidelines that make Just A Minute flow smoothly.
 
 * Not a word until round one. The introduction primes the audience for the show, this is easiest to do with one voice.
 * Good humour is the most important ingredient.
-** Avoid speaking over each other, listening to two voices is hard work.
+* Avoid speaking over each other, listening to two voices is hard work.
 * When the timer is running, one speaker is being featured. Listen closely.
 * There is often some argument following challenges.
   * This is part of the game but KEEP IT FRIENDLY!
-  * The hosts word goes. This isn't just megalomania, it helps to keep the game moving.
+  * The host's word goes. This isn't just megalomania, it helps to keep the game moving.
```

Comment:
Case in point, Parsons Green https://en.wikipedia.org/wiki/Parsons_Green.

To my view, if we say Parson's Green, it seems very much like we are suggesting that the beloved host is, in fact, green, but Parsons Green makes it clear that the Green belongs to the parson.